src/commands/publish/preview/mod.rs

```diff
@@ -19,35 +20,30 @@ struct Preview {
 }
 
 pub fn preview(
-    project: &Project,
     method: Result<HTTPMethod, failure::Error>,
     body: Option<String>,
 ) -> Result<(), failure::Error> {
-    let create_address = "https://cloudflareworkers.com/script";
-
-    let client = http::client();
+    let project = Project::new()?;
```

Comment:
Why aren’t we passing project in anymore?

Reply:
my reason isn't enough to change the overall pattern. swapping back.